src/components/CtaSection/Cta.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../sass/abstracts/variables';
+
+.cta {
+  margin-top: 80px;
```

Comment:
We should keep using rem instead of pixels in most cases.
Target modern browsers, better accessibility and respect user preferences.

Reply:
Okay. Noted.